Led.cpp

```diff
@@ -1,33 +1,33 @@
 #include "Led.h"
 
-Led::Led() : Pin(Pin) {
+Led::Led() : PIN(0) {
 }
 
-Led::Led(unsigned pin) : Pin(pin) {
+Led::Led(unsigned pin) : PIN(pin) {
     //	pinMode(pin, OUTPUT);
 }
 
 void Led::setBrightness(unsigned percent) {
-    //  percent = map(percent, 0, 100, 0, 255);
+    percent *= 2.55;
     //	analogWrite(led, percent);
```

Comment:
If the led is already on then we need to change the active brightness
We do not want to turn it on every time using analogWrite.